package.json

```diff
@@ -108,9 +121,6 @@
         "role": "Editor",
         "icon": "core/static/icons/gristdoc"
       }
-    ],
-    "nsis": {
-      "perMachine": true
```

Comment:
Do you know what this was?

Reply:
According to this [documentation](https://www.electron.build/configuration/nsis), this option does nothing when `oneClick` is `true` (default, we didn't change it), so removed.

Reply:
FYI otherwise this option tells NSIS to skip the installation mode page and always installs Grist Desktop system-wide. This might not be a good idea in restricted environments where user does not have admin privileges, but we now have the portable build so this shouldn't matter much?

Reply:
Pretty sure this was the hard-to-find solution to some gnarly problem, but we shouldn't keep it out of superstition. Someone will shout if it turns out to have some effect that was important, and then we can document the reason better next time :-). Cheers.